File: data-structure-and-algorithm/00-BigO/javascript/O(n^2).js
// Example of O(n^2)
// It runs n * n = n^2 times
function logItems(n) {
     for(let i = 0; i < n; i++) {
         for(let j = 0; j < n; j++) {
             console.log(i, j) 
         }       
     } 
 }
 
 logItems(10)

File: data-structure-and-algorithm/00-BigO/python/O(n^2).py
# Example of O(n^2)
# O(n^2) is also called "Loop within a Loop" time.
# It runs n * n = n^2 times
def print_items(n):
    for i in range(n):
        for j in range(n):
            print(i, j)


print_items(10)
